### PR TITLE
Fixed hardcoded kafka logs, user, group

### PR DESCRIPTION
**Pull request by kcdbaba**

### CODE REVIEW

**Review by vincepii — APPROVED** [review]
I double-checked that variables were already defined in the defaults :+1:

**Review by lhoss — APPROVED** [review]

File: tasks/kafka-cfg.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@
     - kafka-install
 
 - name: "Link alternate logs directory and touch output files"
-  shell: "( test ! -e /usr/local/kafka/logs || rm -rf /usr/local/kafka/logs ) && ln -s /var/log/kafka /usr/local/kafka/logs && touch /var/log/kafka/state-change.log /var/log/kafka/kafkaServer.out && chown kafka:kafka /var/log/kafka/state-change.log /var/log/kafka/kafkaServer.out"
+  shell: "( test ! -e /usr/local/kafka/logs || rm -rf /usr/local/kafka/logs ) && ln -s {{ kafka_log_dir }} /usr/local/kafka/logs && touch {{ kafka_log_dir }}/state-change.log {{ kafka_log_dir }}/kafkaServer.out && chown {{ kafka_user }}:{{ kafka_group }} {{ kafka_log_dir }}/state-change.log {{ kafka_log_dir }}/kafkaServer.out"
```
Comment (thirteen37) [inline tasks/kafka-cfg.yml:53]: Not specific to this PR, but this line looks pretty bad, IMO. What's wrong with `ls -sf`? Why not use Ansible's built-in `file` module? Why are there two different responsibilities (linking log dir and touching files) mashed together?

I've looked at the rest of this repo and it seems to be of mixed quality. Stuff like this (`ansible-kafka/tasks/kafka-install.yml`): 

```
- set_fact: use_systemd=True
  when: init.stdout == 'systemd'
  tags:
    - kafka-install

- set_fact: use_systemd=False
  when: init.stdout != 'systemd'
  tags:
    - kafka-install
```

could be, IMHO, just:

```
- set_fact: use_systemd="{{init.stdout == 'systemd'}}"
  tags:
    - kafka-install
```

And of questionable intent (`ansible-kafka/tasks/system.yml`):

```
  lineinfile: dest=/etc/profile.d/Z99-kafka.sh regexp="^export PATH=.*$" line='export PATH="${PATH}:/usr/local/lib/kafka/bin"' state=present create=yes
```

Comment (kcdbaba) [reply tasks/kafka-cfg.yml:53]: Agreed. Took me a while to decide on which galaxy role. All were broken. This one worked better than the rest.

Comment (lhoss) [reply tasks/kafka-cfg.yml:53]: * that link-wtf task was def. crappy @thirteen37 !
* also totally agree with the 2. point (should def. submit PR !)
* and the last one, yeah  of course no regex/lineinfile magic required  thx to using a *dedicated* profile shell script

Comment (lhoss) [reply tasks/kafka-cfg.yml:53]: but I can still see @fissurezone 's choice for that role .. seeing other hits: https://www.google.ch/search?q=ansible%20kafka&oq=ansible%20kafka
maybe the hp one could be good, just looks quite outdated .. the 3.hit (lloydmeta) is only for centOS ...

**Comment by lhoss** [comment]
> Agreed. Took me a while to decide on which galaxy role. All were broken. This one worked better than the rest.

I'ld just improve it, fill PRs and if they won't get merged then we just keep/maintain our fork , or even re-release  our own role

**Pull request merged**